lib/scsi_defs.h

```diff
@@ -90,5 +92,21 @@ namespace scsi_defs {
     ControlByte control_byte;
   } ABSL_ATTRIBUTE_PACKED; static_assert(sizeof(TestUnitReadyCommand) == kTestUnitReadyCommandSize);
 
+  // SCSI Reference Manual Table 119 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  struct ReadCapacity10Command {
+    OpCode op_code = OpCode::kReadCapacity10;
+    uint8_t reserved_1; // obsolete
+    uint32_t logical_block_address; // obsolete Big endian
+    uint16_t reserved_2;
+    uint8_t reserved_3; // obsolete PMI bit
+    ControlByte control_byte;
+  } ABSL_ATTRIBUTE_PACKED; static_assert(sizeof(ReadCapacity10Command) == kReadCapacity10CommandSize);
+
+  // SCSI Reference Manual Table 120 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  struct ReadCapacity10Data {
+    uint32_t returned_logical_block_address; // Big endian
+    uint32_t block_length; //Big endian
```

Comment:
```suggestion
    uint32_t block_length; // Big endian
```

Also, do we need these comments on every field? If all multibyte fields are big-endian it would be clearer to just document that up front.